src/renderers/shared/fiber/ReactFiberReconciler.js

```diff
@@ -49,18 +49,35 @@ export type HostConfig<T, P, I, TI, PI, C, CX, PL> = {
   getChildHostContext(parentHostContext : CX, type : T) : CX,
   getPublicInstance(instance : I | TI) : PI,
 
-  createInstance(type : T, props : P, rootContainerInstance : C, hostContext : CX, internalInstanceHandle : OpaqueNode) : I,
+  createInstance(type : T,
+                 props : P,
+                 rootContainerInstance : C,
+                 hostContext : CX,
+                 internalInstanceHandle : OpaqueNode) : I,
```

Comment:
Same as [here](https://github.com/facebook/react/pull/8846/files#r98273982) for various methods in this file.